.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
+
+**Describe alternatives you've considered**
+A clear and concise description of any alternative solutions or features you've considered.
+
+**Additional context**
+Add any other context or screenshots about the feature request here.
```

Comment:
I know this is copied from open-liberty repo, but I wonder if we should just go with:
**Describe the feature/functionality you're requesting:**

**Describe why this is important to you:**

**Any additional context:**

Reply:
I think the suggestions are good and soften the tone of the template.  I will engage others to help refine the wording.

Reply:
Hi @dmuelle, can you help advice on how to improve on the template for opening a new feature request for the Liberty Starter?  I would prefer the tone to be welcoming to any new ideas that can improve the Liberty Starter for users.  

See one suggestion [here](https://github.com/OpenLiberty/start.openliberty.io/pull/45#discussion_r808463351).  

My quick attempt is below

```
**Describe the new feature you would like to see be added to the Liberty Starter**  
A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]

**Describe why the new feature is important**  
A clear and concise description of what you want to happen.

**Any additional context**  
Add any other context or screenshots about the feature request here.
```

Reply:
Maybe frame it from the perspective of someone who is excited to contribute rather than frustrated with the current state. 
```
**Describe your idea for new feature to add to the Liberty Starter**  
A clear and concise description of your idea. 
What new capabilities would the feature enable or what problem would it solve? 

**Describe why the new feature is important**  
A clear and concise description of the value of and use case for your suggestion. 
For example, "As a developer/admin/etc. I want to be able to x so that I can y.."

**Any additional context**  
Add any other context or screenshots about the feature request here.
```